package.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Is there any particular reason to delete package.json? We still need it in order to install the Grunt tasks.